api/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM python:3.8-slim
+COPY requirements.txt ./requirements.txt
+RUN pip install -r requirements.txt
+COPY . ./
+
+ENV GOOGLE_APPLICATION_CREDENTIALS='Vincent Van Bot-67b2b35a7d0c.json'
+
+CMD uvicorn api:app --host 0.0.0.0 --port $PORT
```

Comment:
To adhere to standard style, we might want to add a blank new line at the end of each file

Reply:
Ok, will pay attention to it :)